config/deploy.js

```diff
@@ -90,6 +90,7 @@ module.exports = function (deployTarget) {
     ENV['cloudfront'].distribution = 'E1RJJYSB507IYA';
   }
   if (deployTarget === 'development') {
+    ENV.build.environment = 'production';
```

Comment:
i'm confused by this line. why change the build env to production when the deploy target is development?

Reply:
Yeah, that's confusing I agree. The deployment environment is development, however the ember build will be run against production. This ensures that the output is the same as it would be for a production app (minimized, comments stripped out, no tests files) which makes it suitable for testing various things that happen in our build deploys on a semi-annual basis when built for production.